.github/workflows/test.yml

```diff
@@ -7,16 +7,17 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    container: swiftlang/swift:nightly-main-jammy
```

Comment:
The compiler version must be exactly the same as the one used to build the Swift SDK. The compiler revision info is included in GitHub release note.

Reply:
I fixed it, but the Docker Hub doesn't have a history of image digests and I can't find the corresponding logs on ci.swift.org, so it seems difficult to maintain.

https://github.com/kkk669/swift-syntax/compare/713e6fbfbc9857e120fc548e9b60cb6a0b8fbaf0..6db04bc7c37ae0d5c90f87c70e79653bc606c199

Reply:
Ah, good point. I'll feedback to the image CI maintainer

Reply:
Thank you! 🙏